include/iob.h

```diff
@@ -0,0 +1,427 @@
+#ifndef __IOB__
+#define __IOB__ 1
+
+#pragma pack(packed)
+struct iob {
+  union {
+    struct {
+      unsigned int   iobv6chn : 1, /* -   I/O CHAINED BIT SET BY IGG019V6  @ZA56251      */
+                     iobrsv02 : 1, /* RESERVED                                           */
+                     iobrsv03 : 1, /* RESERVED                                           */
+                     iobrsv04 : 1, /* RESERVED                                           */
+                     iobptst  : 1, /* -   NOTE OR POINT OPERATION IS IN PROCESS          */
+                     iobabapp : 1, /* -   ERROR HAS BEEN PROCESSED ONCE BY ABNORMAL-END  */
+                     iobrstch : 1, /* -   RESTART CHANNEL                                */
+                     iobpci   : 1; /* -   SET WHEN A PROGRAM-CONTROLLED INTERRUPTION     */
+      unsigned char  iobrsv05;     /* -     RESERVED                                     */
+      unsigned char  iobcinop;     /* -     OFFSET OF THE LAST I/O COMMAND FOR INPUT     */
+      unsigned char  iobconop;     /* -     OFFSET OF THE LAST I/O COMMAND FOR AN OUTPUT */
+      int            iobcecb;      /* -       EVENT CONTROL BLOCK USED BY BSAM OR QSAM.  */
+      };
+    struct {
+      int            iobvidan; /* VIRTUAL IDAW FOR QSAM, BSAM, BPAM NORMAL  @L3A */
+      struct {
+        int            ioblenrd; /* LENGTH OF BLOCK READ, LBI                 @L5A */
+        } iobvida2;              /* VIRTUAL IDAW WHEN NOT BEGINNING OF BUFFER,@L5C */
+      };
+    struct {
+      struct {
+        unsigned char  iobflag1;    /* -       FLAG BYTE 1                                  */
+        unsigned char  iobflag2;    /* -       FLAG BYTE 2                                  */
+        unsigned char  iobsens0;    /* -       FIRST SENSE BYTE                             */
+        unsigned int   iobs1b0 : 1, /* -   BIT 0 (DEVICE DEPENDENT)                         */
+                       iobs1b1 : 1, /* -   BIT 1 (DEVICE DEPENDENT)                         */
+                       iobs1b2 : 1, /* -   BIT 2 (DEVICE DEPENDENT)                         */
+                       iobs1b3 : 1, /* -   BIT 3 (DEVICE DEPENDENT)                         */
+                       iobs1b4 : 1, /* -   BIT 4 (DEVICE DEPENDENT)                         */
+                       iobs1b5 : 1, /* -   BIT 5 (DEVICE DEPENDENT)                         */
+                       iobs1b6 : 1, /* -   BIT 6 (DEVICE DEPENDENT)                         */
+                       iobs1b7 : 1; /* -   BIT 7 (DEVICE DEPENDENT)                         */
+        struct {
+          unsigned char  iobecbcc;      /* -     COMPLETION CODE FOR AN I/O REQUEST.  THIS CODE */
```

Comment:
The tool that generates these headers doesn't seem to properly handle comments with a newline.  Could you please go back to the macro source and add the missing text?